### ISSUE TITLE
Deserialization issue on python 3.9

### ISSUE THREAD

**Issue by dervan** [Ezibenroc/PyRoaringBitMap#65]
Hello,

I've tried to use pyroaring on python3.9 but it appears there is some issue. I'm testing it on clean virtualenv with pyroaring only:

```
Python 3.9.1 (default, Dec 13 2020, 11:55:53) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyroaring import BitMap
>>> BitMap.deserialize(BitMap([3, 12]).serialize())
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "pyroaring/abstract_bitmap.pxi", line 733, in pyroaring.AbstractBitMap.deserialize
TypeError: type() takes 1 or 3 arguments
```

On older python 3.6 there is no such issue:
```
Python 3.6.12 (default, Nov 14 2020, 12:20:33) 
[GCC 10.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyroaring import BitMap
>>> BitMap.deserialize(BitMap([3, 12]).serialize())
BitMap([3, 12])
```

Do you know what causes an error and if it's easy to fix?

**Comment by lemire** [comment]
No problem with Python 3.8.

**Comment by lemire** [comment]
Issue verified with Python 3.9.

**Comment by lemire** [comment]
Something changed between Python 3.8 and Python 3.9 to break this code.

**Comment by Ezibenroc** [comment]
Hello and thank you for reporting this problem.

To reproduce your issue, I installed Python 3.9 from source and used a virtual environment. What puzzled me is that I could only reproduce the problem by installing pyroaring from pypi (with `pip install pyroaring`).
It never happened when I installed it from source (I tried with release `0.2.9` and with `master`, installed with either `python setup.py bdist_wheel && pip install dist/*` or `python setup.py sdist && pip install dist/*`).

I also added Python 3.9 to travis, the test also succeeded [here](https://travis-ci.org/github/Ezibenroc/PyRoaringBitMap/builds/754466274).

What I eventually found is that both on my laptop and on the current Travis installation, Cython 0.29.21 is installed, whereas to compile the package available on Pypi, Cython 0.29.14 was used.
And indeed, by downgrading Cython to 0.29.14 on my laptop, I was able to reproduce this bug even when installing from source.

I will make a new release compiled with the most recent version of Cython, this should solve your problem.
